examples/gallery/line/vectors.py

```diff
@@ -0,0 +1,63 @@
+"""
+Cartesian, circular, and geographic vectors
+-------------------------------------------
+
+The :meth:`pygmt.Figure.plot` method can plot Cartesian, circular, and geographic vectors.
+The ``style`` parameter controls vector attributes.
```

Comment:
Maybe add a link to https://docs.generic-mapping-tools.org/6.1/plot.html#vector-attributes?

Reply:
We already have an example showing different vector attributes. Perhaps better to link https://www.pygmt.org/dev/gallery/line/vector-heads-tails.html.